samples/html_form_post.js

```diff
@@ -0,0 +1,31 @@
+import http from "k6/http";
+import { check } from "k6";
+
+// Our form data, to be URL-encoded and POSTed
+const form_data = {
+    name: "Test Name",
+    telephone: "123456789",
+    email: "test@example.com",
+    comment: "Hello world!",
+    topping: [
+        'onion',
+        'bacon',
+        'cheese'
+    ]
+};
+
+export default function() {
+    // Passing an object as the data parameter will automatically form-urlencode it
+    let r = http.post("http://httpbin.org/post", form_data);
+
+    // Verify response
+    let j = r.json()["form"];
```

Comment:
Yup, I know, I just thought the code looked cleaner when having an intermediate var to avoid the clutter of doing `r.json()` for every check. I'll change it though, as doing `r.json().name` is clearer than `r.json()["name"]` as you point our below.